Doc/whatsnew/3.10.rst

```diff
@@ -489,17 +489,6 @@ Porting to Python 3.10
   <arg-parsing>` and the :pep:`353`.
   (Contributed by Victor Stinner in :issue:`40943`.)
 
-* Since :c:func:`Py_TYPE()` is changed to the inline static function,
-  ``Py_TYPE(obj) = new_type`` must be replaced with ``Py_SET_TYPE(obj, new_type)``:
-  see :c:func:`Py_SET_TYPE()` (available since Python 3.9). For backward
-  compatibility, this macro can be used::
-
-      #if PY_VERSION_HEX < 0x030900A4
-      #  define Py_SET_TYPE(obj, type) ((Py_TYPE(obj) = (type)), (void)0)
-      #endif
-
-  (Contributed by Dong-hee Na in :issue:`39573`.)
-
 * Since :c:func:`Py_REFCNT()` is changed to the inline static function,
```

Comment:
Also this one.

Reply:
I didn't see any bug report about Py_REFCNT() becoming a static inline function, so I prefer to only change Py_TYPE() and Py_SIZE().

Setting manually the reference counter of an object is a very rare use case. I'm only aware of Cython doing that, and I already updated Cython to use Py_SET_REFCNT() internally.

Reply:
OK.

Reply:
Note: I'm open to reverting Py_REFCNT() as well if the Py_REFCNT() change breaks too many third party Python projects.

Reply:
Ok, I've made a mistake here and I've meant to point out Py_SIZE is not removed. See https://github.com/python/cpython/pull/23375

Reply:
Oh sorry, I misunderstood your comment! Thanks for the PR, it's already merged ;-)